src/IO/AbstractIOHandlerHelper.cpp

```diff
@@ -18,66 +18,73 @@
  * and the GNU Lesser General Public License along with openPMD-api.
  * If not, see <http://www.gnu.org/licenses/>.
  */
-#include <openPMD/IO/ADIOS/ADIOS2IOHandler.hpp>
 #include "openPMD/IO/AbstractIOHandlerHelper.hpp"
-#include "openPMD/IO/DummyIOHandler.hpp"
+
+#include <openPMD/IO/ADIOS/ADIOS2IOHandler.hpp>
```

Comment:
```suggestion
#include "openPMD/IO/ADIOS/ADIOS2IOHandler.hpp"
```
and no extra empty line